resources/helm/dask-gateway/values.yaml

```
@@ -1,3 +1,6 @@
+fullnameOverride: ""
+nameOverride: ""
```

To provide these default values as blank strings vs null values - what makes sense here? Are they both resulting in the same outcomes currently?

```yaml
fullnameOverride:
nameOverride:
```

---

I tried this practically, there was no difference.

Due to a bug in `helm` since 3.2.0+ subchart can't set something to null, let's go for a default value of null instead of a blank string for now as that gives us a bit more flexibility long term.

```suggestion
fullnameOverride:
nameOverride:
```

---

There is no difference between both but that's what most helm charts are doing nowadays:
- https://github.com/bitnami/charts/blob/master/bitnami/etcd/values.yaml#L31
- https://github.com/bitnami/charts/blob/master/bitnami/common/templates/_names.tpl#L21
- https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L23

---

What do you think about trying to keep these values set to default values so that we follow what most people are doing ?
(I have rarely seen values set to null in helm charts)

---

Okay sure!